layouts/basic/modules/Settings/SocialMedia/Logs.tpl

```diff
@@ -0,0 +1,32 @@
+{*<!-- {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} -->*}
```

Comment:
removing the log panel and make a link to the general tab of logs (it is more extensive)